.gitlab-ci.yml

```diff
@@ -177,6 +176,7 @@ benchmark-serverless:
   image: registry.ddbuild.io/ci/serverless-tools:1
   tags: ["arch:amd64"]
   when: on_success
+  timeout: 2h
```

Comment:
Interesting, do we _need_ this? 🤔 This stage depends on the `benchmark-serverless-trigger` job, and I'd be surprised if it _is_ timing out...

Reply:
Yes, you are right, thanks! In this particular case, we don't need it. Thanks!